receiver/vcenterreceiver/scraper_test.go

```diff
@@ -16,6 +16,7 @@ package vcenterreceiver // import github.com/open-telemetry/opentelemetry-collec
 
 import (
 	"context"
+	"go.opentelemetry.io/collector/config/configtls"
```

Comment:
This needs to go into the second import order, other than that, I think it looks good.

Reply:
I have removed "go.opentelemetry.io/collector/config/configtls" in commit [e138022](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15704/commits/e13802243a68167908ed6cba72bcc348f1a39fc3).